app/models/repository_owner/bitbucket.rb

```diff
@@ -72,7 +72,11 @@ def download_members
     def self.create_user(user_hash)
       return if user_hash.nil?
 
-      user_hash = user_hash.to_hash.with_indifferent_access
+      user_hash = user_hash
+        .to_hash
+        .with_indifferent_access
+        .transform_values { |v| v.is_a?(String) ? v.gsub("\u0000", "") : v }
```

Comment:
The amount of repeats suggest we need this logic in a common place. Maybe something like a `StringUtils` utility class or maybe in `RepositoryOwner::Base`?